Formula/swift-sh.rb

```diff
@@ -0,0 +1,21 @@
+class SwiftSh < Formula
+  desc "Scripting with easy zero-conf dependency imports"
+  homepage "https://github.com/mxcl/swift-sh"
+  url "https://github.com/mxcl/swift-sh/archive/1.16.1.tar.gz"
+  sha256 "fb70b075101594b146d4120b52d6132f8b34d3337d4d55ed289c57e4d7808519"
+  head "https://github.com/mxcl/swift-sh.git"
+
+  depends_on :xcode => ["10.0", :build]
```

Comment:
Does it actually depend on xcode or just on swift?

Reply:
@SMillerDev It only depends on swift.

Reply:
@homebrew/core do we want to make it `uses_from_macos "swift"`?

Reply:
@SMillerDev I think leave as-is for now until Homebrew on Linux folks confirm that these type of formulae are working with Swift on Linux.